lib/emojiKeywords.json

```diff
@@ -83,5 +83,8 @@
   "react": "react",
   "apple": "appleinc",
   "cat": "cat",
-  "dog": "dog"
+  "dog": "dog",
+  "code": "male-technologist",
+  "hack": "male-technologist",
```

Comment:
could we make this the non-binary version of the emoji?

Reply:
referring to https://emojipedia.org/technologist/, though im not sure if Slack supports it

if not :goose-honk-technologist: maybe?

Reply:
sure :goose-honk-technologist: is nice! 👍🏻

Reply:
check it now 🧑‍💻 on slack using ":technologist:"

Reply:
check it now 🧑‍💻 "technologist" on slack